### DISCUSSION TITLE
How to change index route to admin route?

### DISCUSSION THREAD

**Discussion by iranandroid** [filamentphp/filament#3311] [Help]
Hi everyone
I'm new in Filament and wanna change index page(\) to admin route?
How to is it?

**Comment by danharrin** [comment]
I don't know if I understand your question, but if you want the admin panel to be mounted to the `your.app/` route instead of `your.app/admin`, change the `path` in config to `''`

**Reply by iranandroid** [reply]
Hi @danharrin 
Yes, you got it right, but I'm new and I don't know where the config file. can you help me?

**Reply by danharrin** [reply]
https://filamentphp.com/docs/2.x/admin/installation#publishing-the-configuration
